src/sonic-config-engine/minigraph.py

```diff
@@ -441,7 +443,11 @@ def parse_meta(meta, hname):
                     erspan_dst = value_group
                 elif name == "DeploymentId":
                     deployment_id = value
-    return syslog_servers, dhcp_servers, ntp_servers, tacacs_servers, mgmt_routes, erspan_dst, deployment_id
+                elif name == "KubernetesMaster":
+                    k8s_master = value
```

Comment:
what if kubemaster is a cluster, is this going to a list or a single server?

Reply:
It is either a single server address or a single HA (Load balanced) IP address, where there are multiple servers behind for HA cluster.
In either case, the node joins to one address only.